lab5/commands/Show.java

```diff
@@ -0,0 +1,22 @@
+package org.example.commands;
+
+import org.example.utils.CollectionManager;
+
+public class Show implements Commands{
+    private final CollectionManager collectionManager;
+
+    public Show(CollectionManager collectionManager) {
+        this.collectionManager = collectionManager;
+    }
+
+    @Override
+    public void execute(String arg) {
+
+    }
+
+    @Override
+    public void execute() {
+        String showResult = collectionManager.show();

```

Comment:
bring out the show logic inside collection manager to here

collection manager should not be concerned about building strings